packages/mdc-radio/index.ts

```diff
@@ -21,13 +21,13 @@
  * THE SOFTWARE.
  */
 
-import MDCComponent from '@material/base/component';
+import {MDCComponent} from '@material/base/component';
 import {EventType, SpecificEventListener} from '@material/base/index';
 import {RippleCapableSurface} from '@material/ripple/index';
 import {MDCRipple, MDCRippleFoundation} from '@material/ripple/index';
 import {MDCSelectionControl} from '@material/selection-control/index';
 
-import MDCRadioFoundation from './foundation';
+import {MDCRadioFoundation} from './foundation';
```

Comment:
This file seems to be missing the usual `export * from` lines that others have been updated to use?

Reply:
Good catch. Done!